Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js

```diff
@@ -32,6 +33,10 @@ export function boundPlane(bounds, origin, p1, p2) {
   vtkMath.cross(v1, v2, n);
   vtkMath.normalize(n);
 
+  // Inflate bounds in order to avoid precision error when cutting cubesource
+  const inflatedBounds = [...bounds];
+  vtkBoundingBox.inflate(inflatedBounds, EPSILON);
```

Comment:
why not `scale` like we discussed ?

Reply:
It doesn't do what we want. The output after scaling is too close to the original bounds.

Reply:
Well... that breaks other test on bound plane. As we modify the bounds, then the expected output data (origin, p1, p2) are not correct. This fix is not the correct solution I think...

Reply:
Indeed, it should be an addition and not a multiplication...
The following should work though...
```
const eps = [...n];
vtkMath.multiplyScalar(eps, EPSILON);
vtkBoundingBox.addBounds(inflatedBounds, bounds[0] + eps[0], bounds[1] + eps[0], ...);
vtkBoundingBox.addBounds(inflatedBounds, bounds[0] - eps[0], bounds[1] - eps[0], ...);
```

Reply:
It's not correct to inflate on all bounds limits: cf [Test boundPlane oriented](https://github.com/Kitware/vtk-js/blob/6aaa03adbfe8a8bc0985a3db03906272ec3c3df5/Sources/Widgets/Widgets3D/ResliceCursorWidget/test/testBoundPlane.js#L38)

Reply:
for this test, `eps` = `[0,0,EPSILON]`, therefore it should not be an issue.

Reply:
For this test, `n != [0, 0, 1]`. We are in an oblique case